lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -329,11 +329,15 @@ sub _set_storage_defaults {
           function => "0x0",
         };
       }
-      elsif ( $store->{"bus"} eq "ide" && !exists $store->{"address"} ) {
+      elsif (
+        ( $store->{"bus"} =~ /\Aide|scsi\Z/ && !exists $store->{"address"} ) )
+      {
+ # The scsi conditional for the bus works around this error during virsh define:
+ # error: internal error: SCSI controller only supports 1 bus
         $store->{"address"} = {
           type       => "drive",
           controller => 0,
-          bus        => 1,
+          bus        => $store->{"bus"} eq "scsi" ? 0 : 1,
```

Comment:
And I believe the single remaining Test::Perl::Critic::Progressive violation is about the double quoted literal, which doesn't need interpolation.

Let's just use single quote.

Reply:
done, thanks.